src/components/WhoWeAre/bios.ts

```diff
@@ -0,0 +1,322 @@
+import McCall from '../../images/who-we-are/board/Board_McCall.jpg';
+import Onigbanjo from '../../images/who-we-are/board/Board_Onigbanjo.jpg';
+import Richardson from '../../images/who-we-are/board/Board_Richardson.jpg';
+import White from '../../images/who-we-are/board/Board_White.jpg';
+import MendezGonzalez from '../../images/who-we-are/board/Board_Mendez Gonzalez.jpeg';
+import Steinrock from '../../images/who-we-are/board/Board_Steinrock.jpg';
+
+//Operations Images
+import Geis from '../../images/who-we-are/operations/Staff_Geis.jpg';
+import McCoy from '../../images/who-we-are/operations/Staff_Kei McCoy.jpg';
+import Zacks from '../../images/who-we-are/operations/Staff_Zacks.jpeg';
+
+//Technical Images
+import Jewell from '../../images/who-we-are/technical/Staff_Jewell.jpg';
+import Nicholson from '../../images/who-we-are/technical/Staff_Nicholson.jpg';
+import Liriano from '../../images/who-we-are/technical/Staff_Liriano.jpg';
+import Voghel from '../../images/who-we-are/technical/Staff_Voghel.png';
+import KendalBrown from '../../images/who-we-are/technical/Staff_Kendal Brown.jpg';
+import Oie from '../../images/who-we-are/technical/Staff_Oie.jpg';
+import Rebollar from '../../images/who-we-are/technical/Staff_Rebollar.jpg';
+import Salemi from '../../images/who-we-are/technical/Staff_Salemi.jpeg';
+import Turner from '../../images/who-we-are/technical/Staff_Turner.png';
+import Volland from '../../images/who-we-are/technical/Staff_Volland.jpg';
+import Wilson from '../../images/who-we-are/technical/Staff_Wilson.jpg';
+
+const bios = {
+  board: [
+    {
+      id: 1,
+      name: 'Anna Schutz',
+      role: 'President',
+      pronouns: 'she/her',
+      image: Schutz,
+      linkedin: 'annaschutz',
+      bio:
+        "Anna Schutz (she/her) - Executive Director holds a BFA in Acting from the University of Illinois Urbana-Champaign and has worked in Chicago as a performer, playwright, and producer for over a decade. She co-founded the storefront theatre Brown Paper Box Co. and served as its Managing Director during its 11 years of operation. \n\nAnna completed her Master of Nonprofit Management at DePaul University where her research was on Representation in Nonprofit Performing Arts Leadership. She served twice as the TA for DePaul's Women in Chicago Theatre class exposing first year students to intersectional feminist theory through field trips to performances around the city.Currently, she oversees daily operations and visits for Undergraduate Admission at DePaul University.There she has the privilege of managing and mentoring students employees, ensuring their professional development.\n\nAs a lover of technology, Anna previously facilitated technical and customer service training during her 6 years with Apple Inc.in its Chicago stores and around the world on its Global Retail Training team.\n\nSunday in the Park with George will never not make her cry. More info at www.annaschutz.com or connect on LinkedIn."
```

Comment:
Rather than redundantly putting the name and pronouns in the bio again, we may want to simply output name, pronouns, and bio separately where we need them together